modules/tensor_mechanics/tests/mohr_coulomb/tests

```diff
@@ -126,6 +133,13 @@
     rel_err = 1.0E-5
     abs_zero = 1.0E-3
   [../]
+  [./uni_axial1_small_strain]
+    type = 'CSVDiff'
+    input = 'uni_axial1_small_strain.i'
+    csvdiff = 'uni_axial1.csv'
+    rel_err = 1.0E-5
+    abs_zero = 1.0E-3
```

Comment:
The problem is: using abs_tol it is easy to get premature convergence and
that's why we frown on it for tests because it can lead to numerical diffs.

I didn't say "DON'T USE abs_tol"... only that it's discouraged in tests
(but obviously... if you know it's appropriate then go for it!)

Derek

On Tue, May 5, 2015 at 6:51 PM Andy Wilkins <notifications@github.com>
wrote:

> In modules/tensor_mechanics/tests/mohr_coulomb/tests
> <https://github.com/idaholab/moose/pull/5030#discussion_r29721612>:
>
> > @@ -126,6 +133,13 @@
> >      rel_err = 1.0E-5
> >      abs_zero = 1.0E-3
> >    [../]
> > +  [./uni_axial1_small_strain]
> > +    type = 'CSVDiff'
> > +    input = 'uni_axial1_small_strain.i'
> > +    csvdiff = 'uni_axial1.csv'
> > +    rel_err = 1.0E-5
> > +    abs_zero = 1.0E-3
>
> OK @friedmud <https://github.com/friedmud> , we'll have to agree to
> disagree on that. I almost always use "abs_tol" in my real simulations as i
> need to ensure that things have converged to a suitable solution. Using
> "rel_tol" means the final result depends upon any crazy ICs/BCs that happen
> to make the initial residual huge or tiny, which is not appropriate in my
> real simulations.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/idaholab/moose/pull/5030/files#r29721612>.
>